README.md

```diff
@@ -1,107 +1,118 @@
 See [action.yml](action.yml) for the full documentation for this action's inputs and outputs.
 
 ```yaml
-
 jobs:
-  build_test_scan_publish:
-    runs-on: (Use our custom runner)
+  build_test_scan:
+    runs-on: eks
     name: CI Pipeline
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 0         
+        fetch-depth: 0
+        
+    - name: Setup
+      uses: variant-inc/lazy-action-setup@feature/init-setup
+        
+    - shell: pwsh
+      run: |
+        Write-Information $RUNNER_WORKDIR_1
+        gci env:* | sort-object name
+        echo "RUNNER_WORKDIR_1=${env:RUNNER_WORKDIR_1}" >> ${env:GITHUB_ENV}
+          
     - name: Lazy action steps
       id: lazy-action
-      uses: variant-inc/lazy-action@master
+      uses: variant-inc/lazy-action-dotnet@feature/create-release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NUGET_TOKEN: ${{ secrets.PKG_READ }}
+        AWS_DEFAULT_REGION: us-east-2
+        GITHUB_USER: variant-inc
       with:
-        src_file_dir: '.'
+        src_file_dir_path: '.'
         dockerfile_dir_path: '.'
-        github_token: ${{ secrets.PKG_READ }}
-        github_owner_token: ${{ secrets.GITHUB_TOKEN }}
-        sonar_token: ${{ secrets.SONAR_TOKEN }}
-        aws_access_key_id: ${{ secrets.DEV_AWS_ACCESS_KEY_ID }}
-        aws_secret_access_key: ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }}
-        docker_repo_name: 'demo-app'
-        docker_image_name: 'demo-repo'
-        nuget_push: 'true'
-        docker_push: 'true'
-        sonar_scan_enabled: 'false'
-
+        ecr_repository: naveen-demo-app/demo-repo
+        nuget_push_enabled: 'true'
+        sonar_scan_in_docker: 'false'
+        nuget_src_project: "src/Variant.ScheduleAdherence.Client/Variant.ScheduleAdherence.Client.csproj"
+        nuget_package_name: 'demo-app'
 ```
 
 ## parameters
 
 ### Input Parameters
 
+| Parameter                    | Default       | Description                                           | Required |
+| ---------------------------- | ------------- | ----------------------------------------------------- | -------- |
+| `src_file_dir_path`          | `.`           | Directory path of the solution file                   | true     |
+| `dockerfile_dir_path`        | `.`           | Directory path of the dockerfile                      | true     |
+| `ecr_repository`             |               | ECR Repository name                                   | true     |
+| `sonar_scan_in_docker`       | "false"       | Is sonar scan running as part of Dockerfile           | false    |
+| `sonar_scan_in_docker_target`|"sonarscan-env"| sonar scan in docker target.                          | false    |
+| `nuget_push_enabled`         | "false"       | Is nuget push enabled to package registry.            | false    |
+| `nuget_package_name`         |               | If you want to give nuget pacakage name.              | false    |
```

Comment:
For `nuget_package_name` & `nuget_src_project`, add "Used only if `nuget_push_enabled` is true.

Also, `nuget_package_name` - If provided, push the nuget package with this name. By default, it will be the name of the project

Reply:
Updated the description.